### PR TITLE
Update institut-francais-darcheologie-orientale.csl

### PR DESCRIPTION
**Pull request by NicolasSouchon**
Replace institut-francais-darcheologie-orientale.csl by a new CSL style corresponding to the updated version of the Institute français d'archéologie orientale’s standards (available here: https://www.ifao.egnet.net/uploads/publications/normes/IFAO_publications_normes_bibliographiques_pub_egypto_2020_fr.pdf)

### CODE REVIEW

File: institut-francais-darcheologie-orientale.csl [hunk 1]
```diff
@@ -1,337 +1,588 @@
...
+    <link href="https://www.ifao.egnet.net/uploads/publications/normes/IFAO_publications_normes_bibliographiques_pub_egypto_2020_fr.pdf" rel="documentation"/>
     <author>
-      <name>Alexandre Vincent</name>
-      <email>alexandre.vincent@univ-poitiers.fr</email>
-    </author>
-    <author>
-      <name>Nicolas Monteix</name>
-      <email>nicolas.monteix@univ-rouen.fr</email>
+      <name>Nicolas Souchon</name>
+      <email>souchon.nicolas.ns@gmail.com</email>
```
Comment (rmzelle) [inline institut-francais-darcheologie-orientale.csl:12]: @NicolasSouchon, thanks. I see you removed the "template" link and original style authors. Did you create your version from scratch, or did you based it on the existing IFAO style, or another CSL style?

Comment (NicolasSouchon) [reply institut-francais-darcheologie-orientale.csl:12]: Yes, this style was created from an empty base using Visual CSL Editor. But I should add a link between this style and the [English version](https://github.com/citation-style-language/styles/pull/4870)

Comment (rmzelle) [reply institut-francais-darcheologie-orientale.csl:12]: I'm a little bit surprised you went for a complete rewrite. Did the IFAO change its citation format drastically recently? Our IFAO style was only contributed in April, so it's not very old.

@NMonteix, do you know if the style guide has recently changed, and whether this style would be an improvement over your version we currently have in the repository? See also #4870 for the English version of this new style.

**Comment by NMonteix** [comment]
@rmzelle

Because of internal communication problems within the IFAO, encoding 
style has been done in two directions, one (contribution in April) with 
the older style, the other being with possible drastic changes I am not 
aware of.


Le 26/06/2020 à 05:59, Rintze M. Zelle a écrit :

**Comment by rmzelle** [comment]
> Because of internal communication problems within the IFAO, encoding style has been done in two directions, one (contribution in April) with the older style, the other being with possible drastic changes I am not aware of.

Okay, I might just accept this new version, then.

**Comment by NicolasSouchon** [comment]
Dear all,

I saw that this issue and [this one](https://github.com/citation-style-language/styles/pull/4870) will be automatically closed soon for inactivity. Is this normal?

**Comment by rmzelle** [comment]
Thanks! Apologies for the wait.

**Pull request merged**